File: VB/CustomReportClass.Module.Win/WinModule.vb
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace CustomReportClass.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class CustomReportClassWindowsFormsModule
		Inherits ModuleBase

		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace

File: CS/CustomReportClass.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace CustomReportClass.Module.Win {
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class CustomReportClassWindowsFormsModule : ModuleBase {
        public CustomReportClassWindowsFormsModule() {
            InitializeComponent();
        }
    }
}
